priv/repo/migrations/20160802030642_create_note.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Notox.Repo.Migrations.CreateNote do
+  use Ecto.Migration
+
+  def change do
+    create table(:notes) do
+      add :title, :string
+      add :description, :string
+
+      timestamps
```

Comment:
`timestamps null: false` doesn't work. The option isn't listed in the docs, so I'm assuming you got carried away with the `null: false` omissions. :smile:

[`timestamps/1`](https://hexdocs.pm/ecto/Ecto.Schema.html#timestamps/1)